features/polyPinion/src/components/buttons/SubmitSurveyButton.tsx

```diff
@@ -7,7 +7,8 @@ import submitAnswers from "../../server/answer-submission";
 import { useHistory } from "react-router-dom";
 
 export default function SubmitSurveyButton({ title, onStart = () => {}, onFinished = () => {} }) {
-    const { markQuestionaireSubmitted } = useContext(QuestionnaireListContext);
+    const { markQuestionnaireSubmitted: markQuestionnaireSubmitted } =
```

Comment:
Did ESlint actually complain about the typo there? :D Interesting. But a typo it is - any chance we can fix it in `QuestionnaireListContext` instead?

Reply:
ah no this was mostly on the go as I bumped there. I know it's out of context but thought cannot harm that much?! 
QuestionnaireListContext has been changed as well 👍🏽

Reply:
Hm, looked to different keys to me - if it's the exact same keys the syntax you used is just redundant, no? Couldn't it just be:

```
const { markQuestionnaireSubmitted } = useContext(QuestionnaireListContext);
```

Reply:
ah I now see what you meant. :thumbs-up okayy :)